src/controller/java/src/matter/onboardingpayload/OnboardingPayload.kt

```diff
@@ -115,15 +115,8 @@ class OnboardingPayload(
    */
   var setupPinCode: Long = 0
 ) {
-  var optionalQRCodeInfo: HashMap<Int, OptionalQRCodeInfo>
```

Comment:
Removed optionalQRCodeInfo. Don't believe this exists in the iOS setupPayload

Reply:
you moved optionalQRCodeInfo to be initialized directly where it is declared. optionalQRCodeInfo is not removed

Reply:
Just added optionalQRCodeInfo back since one of the tests was failing. I believe it mentioned ChipToolActivity is using it